proxy/src/http/user.rs

```diff
@@ -254,7 +254,7 @@ mod handler {
     /// Transfer funds to the given `recipient`.
     pub async fn transfer<R>(
         ctx: http::Ctx<R>,
-        _handle: registry::Id,
+        _handle: String,
```

Comment:
Why are we changing this from a `registry::Id` to a `String`?

Reply:
I changed it so the API playground works again. It doesn't convert obkects correctly. @NunoAlexandre Did you find another way around this?

Reply:
@MeBrei nop. The issue with using the `Id` type is that `warp` will try and use its default deserializer to parse a request and match it to and endpoint/handler. If it fails to do that, the whole thing fails with a 500 instead of a 400. I.e, say we do `users/123/transfer`. We know this should fail because `123` is not a valid `Id`. However, what warp sees is that there is no endpoint that expects a number so it fails completely. So yeah, better work with primitive numbers and avoid this implicit stuff taking place underneath and taking the control away from us.

Reply:
addressed https://github.com/radicle-dev/radicle-upstream/pull/712/commits/059b135f36003ee31ca3b3d725e96732a61fe2bc